src/replays/_replayswidget.py

```diff
@@ -322,7 +347,9 @@ def updatemyTree(self):
 
                     # Parse replayinfo into data
                     if item.info.get('complete', False):
-                        t = time.localtime(item.info.get('launched_at', item.info.get('game_time', time.time())))
+                        # 'game_time' is 'launched_at' for older replays (if that fails maybe 'game_end' is there)
```

Comment:
I once downloaded a replay from forums (4856559) that when put into the replays folder, always shows on the top. It seems to be parsed properly, but this part of the code makes it always appear on top, not in the 'incomplete replays' bucket. Maybe moving it into 'incomplete' bucket is a good idea if we can't find the start date.